CONTRIBUTING.md

```diff
@@ -26,9 +26,9 @@ in order to stop on first error. Commands shall not source each other. The ony
 way to share code is to source other bash files outside of `lib/commands`.
 
 
-We are not strictly POSIX complaint, not seek to have it as a requirement
-but our CI test should at least run on Linux and MacOS. When possible, prefer
-commands and options that work on most platforms.
+We are not strictly POSIX complaint, nor seek to have it as a requirement, but
+we do want things to work on both Linux and MacOS (our CI runs on both OSes).
+When possible, prefer commands and options that work on most platforms.
```

Comment:
This was the part I was least sure about. I think there is a missing word or typo in the original paragraph, and it's making it hard for me to understand what it exactly it was trying to say.

Reply:
you got it perfect. I was trying to say that we might be as cross-platform as possible, while we don't *need* to be strictly POSIX complaint.